README.md

```diff
@@ -91,5 +91,15 @@ docker run -p 8001:8001 -e SPRING_DATASOURCE_PASSWORD=yourPasswordHere --rm mod-
 
 ### Issue tracker
 
-The project use this GitHub [issue tracker](https://github.com/library-data-platform/mod-ldp/issues).
+The project uses this GitHub [issue tracker](https://github.com/library-data-platform/mod-ldp/issues).
+
+### Code analysis
+
+[SonarQube analysis](https://sonarcloud.io/dashboard?id=org.folio%3Amod-ldp).
+
+### Download and configuration
+
+The built artifacts for this module are available.
+See [configuration](https://dev.folio.org/download/artifacts) for repository access,
+and the [Docker image](https://hub.docker.com/r/folioorg/mod-ldp/).
```

Comment:
Note that mod-ldp and ui-ldp will target both FOLIO and ReShare.  It is unclear to me how much FOLIO-specific CI should be in these repositories, if any; but I think at least we need to label it clearly as FOLIO-specific content.

Reply:
I made some clarificiation via PR #21